### ISSUE TITLE
Query not returning results when looking for a version number

### ISSUE THREAD

**Issue by jacargentina** [fergiemcdowall/search-index#180]
On the following code:

````javascript
var searchIndex = require('search-index');

var si = searchIndex({
  indexPath: 'search_index',
  logLevel: 'debug'
});

var example = [{
  id: 'testing',
  content: 'Nexion Smart ERP 14.2.1.0\n\nRelease de ejemplo'
}];

si.add(example, {}, function(err) {
  if (err) {
    console.log('ERROR:', err);
  }

  si.search({
    query: {
      '*': ['ERP']
    },
    teaser: 'content'
  }, function(err, results) {
    if (err) {
      console.log('ERROR:', err);
    } else {
      console.log('RESULTS ERP:', results);
    }

    si.search({
      query: {
        '*': ['14.2.1.0']
      },
      teaser: 'content'
    }, function(err, results) {
      if (err) {
        console.log('ERROR:', err);
      } else {
        console.log('RESULTS 14.2.1.0:', results);
      }
    });

  });
});
````

Why the search (2nd one) for `14.2.1.0` is not returning results? May be there is some important i'm missing (i've looked on docs, issues, etc. but no luck. Last resource: post here!)

Cheers!

**Comment by fergiemcdowall** [comment]
Hmmm, this is to do with the tokenizer- it is splitting the version number on the ".", and then removing the numbers 2, 1 and 0, since they are defined as stopwords.

**Comment by fergiemcdowall** [comment]
I wonder how best to deal with this...?

**Comment by fergiemcdowall** [comment]
This is now fixed in the latest version of search index (0.6.10) so you need to npm install search-index again

Basically you need to pass a custom separator in `.add`. The separator is a [string split regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)

For your document you want something like this `{separator:/[ (\n)]+/}`

Your test script is now:

```javascript
var searchIndex = require('search-index');

var si = searchIndex({
  indexPath: 'search_index',
  logLevel: 'warn'
});

var example = [{
  id: 'testing',
  content: 'Nexion Smart ERP 14.2.1.0\n\nRelease de ejemplo'
}];

var ops = {separator:/[ (\n)]+/}
si.add(example, ops, function(err) {
  if (err) {
    console.log('ERROR:', err);
  }

  si.search({
    query: {
      '*': ['ERP']
    },
    teaser: 'content'
  }, function(err, results) {
    if (err) {
      console.log('ERROR:', err);
    } else {
      console.log('RESULTS ERP:', results);
    }

    si.search({
      query: {
        '*': ['14.2.1.0']
      },
      teaser: 'content'
    }, function(err, results) {
      if (err) {
        console.log('ERROR:', err);
      } else {
        console.log('RESULTS 14.2.1.0:', results);
      }
    });

  });
});

```

Which should work as expected

**Comment by jacargentina** [comment]
Yes! Thanks a lot!
